src/Trace/Exporter/ZipkinExporter.php

```diff
@@ -150,9 +143,6 @@ public function convertSpans(TracerInterface $tracer, $headers = null)
                 'localEndpoint' => $localEndpoint,
                 'tags' => $span->attributes()
             ];
-            if (array_key_exists($span->kind(), $kindMap)) {
-                $zipkinSpan['kind'] = $kindMap[$span->kind()];
-            }
```

Comment:
Does it mean the ZipkinExporter will loose the ability of setting the kind?

Reply:
Yes, kind is not present in the OpenCensus specification. Stackdriver Trace v2 also removed the kind field. It's possible to overload attributes and try and detect the kind from an attribute. This is a cross-language potential issue, so we'll regroup on this later.

Reply:
How it will affects the user experience of ZipKinExporter? If it's minor, let's go for it. If it's disastrous, can we think of how to mitigate it?

Reply:
It's pretty minor. Essentially, it's a special named attribute for the span and is optional.